cli/generators/crud/templates/appService.js

```diff
@@ -0,0 +1,81 @@
+import fetch from 'isomorphic-fetch';
+import * as endpoints from './apiEndpoints';
+
+class <%= ucName %>Service {
+  static load<%= pluralizedUcName %>() {
+    const request = new Request(`${endpoints.BASE_URL}${endpoints.GET_<%= pluralizedName.toUpperCase() %>}`, {
+      method: 'GET'
+    });
+
+    return fetch(request).then(response => {
+      return response.json();
+    }).catch(error => {
+      return error;
+    });
```

Comment:
Yes.. that could be simplify as @AlphaGit says. And also if you need to check the statusCodes for example in the actions layer, you can only return: `return fetch(request);` and grab the data on the actions file. For example checking *response.status* or getting the response data body with *response.json()* function.